source/sync/configure/sync-settings.txt

```diff
@@ -61,15 +61,48 @@ update the backend schema by changing the client model.
 To learn more about how the Realm Object Schemas map to the App Services Schemas
 when using Development Mode, refer to :ref:`Data Model Mapping <sync-data-model-mapping>`.
 
-For more information about modifying synced object schemas, including how to
-handle breaking changes, see: :ref:`<synced-schema-overview>`.
+For more information about modifying synced object schemas, see:
+:ref:`<synced-schema-overview>`.
 
 .. important:: Disable Development Mode for Production Apps
    
    Development mode is a development utility that is not suitable for 
    production use. Make sure that you turn off Development Mode before you 
    make your app accessible in a production environment.
 
+.. _dev-mode-breaking-changes:
+
+Breaking Changes
+````````````````
+
+*Applies to App Services Apps created after [ADD RELEASE DATE].* 
+
+App Services Apps in Development Mode that were created after [ADD RELEASE DATE]
+can make :ref:`breaking changes <breaking-change-quick-reference>` to synced
+object schemas from client code.
+
+.. note:: Apps created before [ADD RELEASE DATE]
+   
+   For Apps created before [ADD RELEASE DATE], you need to update your
+   object schema in the App Services UI. For details, see
+   :ref:`<synced-schema-overview>`
+
+To make a breaking change from client code:
+
+#. Change your local object model.
+#. Delete your local realm and data.
+#. Open a new realm with your updated object model.
```

Comment:
Instead of saying `"new"` should it just be `"Open the realm..."`? It is technically a new realm under the hood, but I think as far as the users are concerned it's the same realm with just a different schema. The "new" terminology could potentially lead to confusion (e.g. "what happens to the old realm" "how do I open a new realm").

I think there is also another step of "Run your client application" to ensure that it connects and applies the changes to the server schema.

Reply:
Hmm. I see what you mean. I think just "Open a realm" would work. Though it may be good to make it clear that it is, in fact, not the same realm. This could help reinforce the locally-destructive nature of the process.

And good point about adding another step there. I'll update!

Reply:
Open a new realm also works. I just don't want users to think they have to do something differently. They really just need to do `Realm.delete(config)` before the existing `Realm.open()` they already had.